Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:10.18.0-buster AS build-env
+RUN apt-get install -y git 
+RUN mkdir /build
+RUN cd /build && git clone https://github.com/RTradeLtd/ens-app.git
```

Comment:
TODO FOR ENS TEAM: This to be changed into `ensdomains` once merged.

Reply:
Also, is Docker necessary? I am trying to manually run it locally but getting the following error when trying to run our test dataset

```
~/.../ens/ens-app (pr/581)$yarn preTest
yarn run v1.15.2
$ node -r esm src/testing-utils/preTest.js
(node:70418) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'getId' of undefined
    at setupWeb3 (/Users/makoto/work/ens/ens-app/src/testing-utils/preTest.js:13:40)
    at init (/Users/makoto/work/ens/ens-app/src/testing-utils/preTest.js:38:28)
    at Object.<anonymous> (/Users/makoto/work/ens/ens-app/src/testing-utils/preTest.js:80:1)
    at Generator.next (<anonymous>)
(node:70418) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
✨  Done in 4.84s.

```

This is also failing on our CI build https://travis-ci.org/github/ensdomains/ens-app/builds/666226825

Reply:
Yea I can change it and push the change, it was only set to our end point for dev work.

Naw it's not necessary and can be removed, it was just easier to be able to test stuff locally and push it to our dev environment using  docker images.

Is that error coming from the docker container or from your host OS that you built the container on? I'll have Dallas look